bundles/create_openhab_binding_skeleton.cmd

```diff
@@ -0,0 +1,40 @@
+@echo off
+
+SETLOCAL
+SET ARGC=0
+
+FOR %%x IN (%*) DO SET /A ARGC+=1
+
+IF %ARGC% NEQ 3 (
+	echo Usage: %0 BindingIdInCamelCase Author GithubUser
+	exit /B 1
+)
+
+SET OpenhabVersion="2.5.0-SNAPSHOT"
+
+SET BindingIdInCamelCase=%~1
+SET BindingIdInLowerCase=%BindingIdInCamelCase%
+SET Author=%~2
+SET GithubUser=%~3
+
+call :LoCase BindingIdInLowerCase
+
+call mvn -s ../archetype-settings.xml archetype:generate -N -DarchetypeGroupId=org.openhab.core -DarchetypeArtifactId=org.openhab.core.tools.archetypes.binding -DarchetypeVersion=%OpenhabVersion% -DgroupId=org.openhab.binding -DartifactId=org.openhab.binding.%BindingIdInLowerCase% -Dpackage=org.openhab.binding.%BindingIdInLowerCase% -Dversion=%OpenhabVersion% -DbindingId=%BindingIdInLowerCase% -DbindingIdCamelCase=%BindingIdInCamelCase% -DvendorName=openHAB -Dnamespace=org.openhab -Dauthor="%Author%" -DgithubUser="%GithubUser%"

```

Comment:
```suggestion
call mvn -s ../addons/archetype-settings.xml archetype:generate -N -DarchetypeGroupId=org.openhab.core -DarchetypeArtifactId=org.openhab.core.tools.archetypes.binding -DarchetypeVersion=%OpenhabVersion% -DgroupId=org.openhab.binding -DartifactId=org.openhab.binding.%BindingIdInLowerCase% -Dpackage=org.openhab.binding.%BindingIdInLowerCase% -Dversion=%OpenhabVersion% -DbindingId=%BindingIdInLowerCase% -DbindingIdCamelCase=%BindingIdInCamelCase% -DvendorName=openHAB -Dnamespace=org.openhab -Dauthor="%Author%" -DgithubUser="%GithubUser%"
```

Since we move evrything out of addons, shouldn't we move the `archetype-settings.xml` to `bundles`, too?

Reply:
> shouldn't we move the archetype-settings.xml to bundles, too?

Yes, indeed. Makes sense.

Reply:
I've moved `archetype-settings.xml` to the bundles map.